mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule ServerSentEventStage.MixProject do
     [
       app: :server_sent_event_stage,
       version: "1.0.0",
-      elixir: "~> 1.6 or ~> 1.7 or ~> 1.8 or ~> 1.9",
+      elixir: "~> 1.7 or ~> 1.8 or ~> 1.9",
```

Comment:
Isn't this equivalent to just `"~> 1.7"`?

Reply:
I suppose you're right :)